docs/platform/object-storage/how-to-access-objects-with-linode-object-storage/index.md

```diff
@@ -0,0 +1,75 @@
+{{</ note >}}
+
+Object Storage gives each object a unique URL with which you can access your data. An object can be publicly accessible, or you can set it to be private and only visible to you. This makes Object Storage great for sharing and storing unstructured data like images, documents, archives, streaming media assets, and file backups, and the amount of data you store can range from small collections of files up to massive libraries of information.
+
+In this guide you will learn how to access the objects you have stored in Linode's Object Storage for:
+
+- [Object URLs](#object-urls)
+
+- [Signed URLs](#signed-urls)
+
+- [Websites](#websites)
+
+## Before You Begin
+
+To learn how to enable Object Storage, see the [How to Use Object Storage](/docs/platform/object-storage/how-to-use-object-storage/) guide.
+
+Object Storage is similar to a subscription service. **Once enabled, you will be billed at the flat rate regardless of whether or not there are active buckets on your account.** [Cancelling Object Storage](/docs/platform/object-storage/how-to-use-object-storage/#cancel-object-storage) will stop billing for this flat rate.
+
+In all Object Storage URLs the cluster where your bucket is hosted is a part of the URL string.
+
+{{< content "object-storage-cluster-shortguide" >}}
+
+## Object URLs
+
+Objects stored in Linode object storage are generally accessible using this format:
+
+    http://my-example-bucket.us-east-1.linodeobjects.com/example.txt
+
+- Replace `my-example-bucket` with your bucket name, `us-east-1` with the cluster where your bucket is hosted, and `example.txt` with the object you wish to access.
+
+- This assumes that the object is publicly accessible. For more on object permissions, see the [How to Use Object Storage](/docs/platform/object-storage/how-to-use-object-storage/) guide.
+
+## Signed URLs
+
+Creating a **signed URL** will allow you to create a link to objects with limited permissions and a time limit to access them. Signed URLs have a similar format:
+
+    http://my-example-bucket.us-east-1.linodeobjects.com/example.txt?AWSAccessKeyId=YOUROBJECTSTORAGEACCESSKEY&Expires=1579725476&Signature=rAnDomKeySigNAtuRe
+
+- This is returned when you use a tool like the [Linode CLI](/docs/platform/object-storage/how-to-use-object-storage/#create-a-signed-url-with-the-cli) or [s3cmd](/docs/platform/object-storage/how-to-use-object-storage/#create-a-signed-url-with-s3cmd) to generate a signed URL.
+
+- `my-example-bucket` will be your bucket name, `us-east-1` will be the cluster where your bucket is hosted, `example.txt` will be the object you are giving access to.
+
+- The rest of the URL are the parts that make this URL public for a limited amount of time.
+
+## Websites
+
+Static sites are accessed from different URLs than other objects in your Object Storage bucket. Static sites are available at the `website-us-east-1` subdomain. Using `my-example-bucket` as an example this would be:
```

Comment:
"Static sites are served from URLs that are different than the standard URLs you would normally use to access objects."
And maybe open up the language to say that you simply need to append "website-" to the beginning of a cluster to access the website, as this will be the same for each cluster we roll out.

Reply:
updated and clarified with the prepended website- language a bit